buildSrc/src/main/kotlin/Constants.kt

```diff
@@ -1,3 +1,3 @@
 object BuildConfigConstants {
-    const val IMAGE_DIFFERENCE_THRESHOLD = 0.1f
+    const val IMAGE_DIFFERENCE_THRESHOLD = 1f
```

Comment:
Isn't that threshold 100%?

Reply:
I'm testing with CI/CD so it's not final yet, I'm looking for the closest deviation. Pls wait hehe

Reply:
No rush^